eventhubs/client/azure-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubClientBuilder.java

```diff
@@ -128,11 +151,11 @@ public EventHubClientBuilder connectionString(String connectionString, String ev
     /**
      * Sets the configuration store that is used during construction of the service client.
      *
-     * The default configuration store is a clone of the {@link ConfigurationManager#getConfiguration() global
-     * configuration store}, use {@link Configuration#NONE} to bypass using configuration settings during construction.
+     * If none is specified, the default configuration store is used to configure the {@link EventHubClient}. Use {@link
```

Comment:
nit:  This may read a bit awkwardly.   Maybe consider something like "If not specified, ..."